contracts/AxelarGateway.sol

```diff
@@ -50,13 +50,21 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
     // solhint-disable-next-line var-name-mixedcase
     address internal immutable AUTH_MODULE;
     // solhint-disable-next-line var-name-mixedcase
+    address internal immutable GOVERNANCE_MODULE;
```

Comment:
So to allow a change of governance modules, there needs to be an upgrade to a new instance of the implementation with a difference governance module account. Not as explicit but it's more gas efficient so works i think